http/form/src/main/java/org/wildfly/security/http/form/FormMechanismFactory.java

```diff
@@ -51,6 +51,9 @@ public FormMechanismFactory() {
     public FormMechanismFactory(final Provider provider) {
     }
 
+    public FormMechanismFactory(final Provider... providers) {
+    }
```

Comment:
Where does this change come from?  It seems unrelated to the Jira being addressed - is GitHub showing something it shouldn't or has this come in from a rebase?